demosplan/DemosPlanCoreBundle/Logic/Statement/StatementService.php

```diff
@@ -560,7 +560,11 @@ public function submitDraftStatement(
             $this->statementAttributeRepository->copyStatementAttributes($draftStatement, $originalStatement);
 
             // Create a statement copy for the assessment table
-            $assessableStatement = $this->statementCopier->copyStatementObjectWithinProcedure($originalStatement, false);
+            $assessableStatement = $this->statementCopier->copyStatementObjectWithinProcedureWithRelatedFiles(
```

Comment:
What happens If draftstatement has file?
It's not better we have a condition for this situation. Like:
`if ($draftStatement->getFiles()){$assessableStatement = $this->statementCopier->copyStatementObjectWithinProcedureWithRelatedFiles(...)
}else{$assessableStatement = $this->statementCopier->copyStatementObjectWithinProcedure(...)}`

Reply:
I do not see any reason to use `copyStatementObjectWithinProcedureWithRelatedFiles()` for both cases (with file and without file) in this case of creation a new statement and his original.
If there is no file the loop over the files will just skipped.